x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/utils/append_on_save_query_params.ts

```diff
@@ -30,7 +30,11 @@ export function appendOnSaveQueryParamsToPath({
   paramsToApply.forEach((paramName) => {
     const paramOptions = mappingOptions[paramName];
     if (paramOptions) {
-      const [paramKey, paramValue] = createQueryParam(paramName, paramOptions, policy.policy_id);
+      const [paramKey, paramValue] = createQueryParam(
+        paramName,
+        paramOptions,
+        policy.policy_ids[0] // TODO handle multiple
```

Comment:
not in this pr, I have to discuss what the behaviour should be in case of multiple agent policies, we can't navigate the page to more than one agent policies at once